<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Caracteristicas</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>21</y>
          <width>211</width>
          <height>95</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Codigo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="codigo">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Descrição</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="descricao">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Garantia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="garantia">
             <property name="font">
              <font>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="text">
              <string>Meses</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>228</width>
          <height>63</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Forncedor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fornecedor">
             <property name="styleSheet">
              <string notr="true">background-color:  rgb(200,200,200);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adicionar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="recursos.qrc">
               <normaloff>:/img/img/icons8-plus.svg</normaloff>:/img/img/icons8-plus.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Data Entrada</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="data_entrada">
             <property name="font">
              <font>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="title">
        <string>Tipo</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="2" colspan="2">
          <widget class="QCheckBox" name="infraestrutura">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Infra Estrutura</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="outros">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Outros</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="vigilancia">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Vigilancia</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="informatica">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Informática</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="escritorio">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Escritório</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
